src/Exchange.ts

```diff
@@ -142,23 +153,23 @@ export default class Exchange extends Contract {
             (res, order) => {
                 const bn_amount = new BigNumber(order[3]);
                 if (!bn_amount.eq(0)) {
-                    const parsed = {
+                    const parsed: Parsed = {
                         id: parseInt(order[0], 10),
-                        maker: "0x" + new BigNumber(order[1]).toString(16).padStart(40, "0"), // leading 0s if address starts with 0
+                        maker: `0x${new BigNumber(order[1]).toString(16).padStart(40, "0")}`, // leading 0s if address starts with 0
                         bn_price: new BigNumber(order[2]),
                         bn_amount,
-                        price: 0,
+                        price: new BigNumber(0),
```

Comment:
@rszaloki : just spotted this after PR merge. Why did you change it? I'm not sure if it won't break ratesfeeder... But what's the point to handle price in bignumber when it's int32?